AnagramTS_/src/anagram.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { wordToCheck } from "./anagram";
+import { wordToCheck } from "./app";
 
```

Comment:
Which unit are we testing? The app unit or the anagram unit? This test should test the anagrams, not the app that uses them.